### PR TITLE
OSCI: Onboard UI e2e

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

This PR adds support to run UI e2e tests under OpenShift CI.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

CI is sufficient - [prow rehearse job
](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29081/rehearse-29081-pull-ci-stackrox-stackrox-gavin-OSCI-onboard-ui-e2e-gke-ui-e2e-tests/1532761408666603520)

### CODE REVIEW

File: tests/e2e/lib.sh
Comment (gavin-stackrox) [reply tests/e2e/lib.sh:42]: @pedrottimark yeah. it looks like I missed a few and/or things have changed since I opened the PR. I will rebase and check the list.

Comment (pedrottimark) [reply tests/e2e/lib.sh:44]: Follow up question. Which tests will call this function? Depending on your viewpoint, it was a bug or a feature that CircleCi config.yml required or allowed different feature flags for different tests, for example:

https://github.com/stackrox/stackrox/blob/master/.circleci/config.yml#L3372-L3374

Comment (gavin-stackrox) [reply tests/e2e/lib.sh:44]: @house-d feel free to approve if you think the code looks ok. I'm going to also wait for Marks review.

@pedrottimark I was leaning towards calling the existing behavior a bug, particularly how env settings were duplicated when new cluster flavors were added. But I can also see the benefit of clearly calling out flags close to the test that uses them. So maybe that is the way to go with feature flags. e.g. I could see setting the desired `ROX_` feature flags here: https://github.com/stackrox/stackrox/blob/f117fadd0f19eaa2e631b97596d3e85bb42c0238/.openshift-ci/gke_ui_e2e_test.py#L20 or maybe even before https://github.com/stackrox/stackrox/blob/f117fadd0f19eaa2e631b97596d3e85bb42c0238/tests/e2e/run-ui-e2e.sh#L31.

Comment (pedrottimark) [reply tests/e2e/lib.sh:44]: Yeah, I was leaning in the same direction that it is a bug, having seen some feature flags in ui but not api recently.

Got for it and if follow up discussion suggests separation by test is needed, then you can refactor later.

**Review by pedrottimark — APPROVED** [review]

**Pull request merged**
